### DISCUSSION TITLE
Configuration using `*.sql` file

### DISCUSSION THREAD

**Discussion by r-priyam** [ts-safeql/safeql#86] [Q&A] [answered]
Hello, I, or say many people when using Postgres with docker use the `init.sql` way to initialize the database and migrations. Is there any way to configure it with these file(s) instead of the connection itself?

**Comment by Newbie012** [comment, marked as answer]
Hi!

(A docker rookie here)

Are you looking for [migrationsDir](https://safeql.dev/api/index.html#connections-migrationsdir)? If specified, it will create a new shadow database and run the migrations on it.

**Reply by Newbie012** [reply]
And is that's not the case, and you want to manually initialize the database (with migrations), maybe you could do it before running ESLint and then specify a `databaseUrl`?

**Reply by r-priyam** [reply]
Apologies for the late reply, @Newbie012. I just tested it out with the `migrationsDir` and it worked as expected. Thanks!
